tsconfig.json

```diff
@@ -3,16 +3,20 @@
     "target": "ES2022" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
     "module": "ESNext" /* Specify what module code is generated. */,
     "moduleResolution": "Bundler",
+    "moduleDetection": "force",
```

Comment:
https://www.totaltypescript.com/tsconfig-cheat-sheet

Reply:
We love Matt Pocock 🫡